src/test/java/org/folio/edge/EdgeInnReachApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package org.folio.edge;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class EdgeInnReachApplicationTests {
```

Comment:
You can remove this test and exclude class EdgeInnReachApplication   in the pom.xml in the section `sonar.exclusions`

Reply:
If I understand correctly you meant just remove test for spring app start point and configure property name to skip sonar checking for start point of this module. Correct?